src/hooks/usePreloader.tsx

```diff
@@ -0,0 +1,17 @@
+import { useEffect } from 'react';
+
+const usePreloader = (isLoaded: boolean) => {
+  const checkData = (): void => {
+    const preloader = document.querySelector('.preloader');
+
+    if(isLoaded && preloader) {
+      preloader.remove();
+    }
+  }
+
+  useEffect(() => {
+    checkData();
+  })
+}
+
+export default usePreloader;
```

Comment:
pls, add extra line in the end

Reply:
There is, I don't know why it is hidden here.